demos/benchmark/python/README.md

```diff
@@ -2,20 +2,15 @@
 
 ## Introduction
 
-The benchmark client introduced in this directory is written in Python 3. It is recommended to use the benchmark client as a
-docker container. Prior to transmission, the client downloads metadata from the server, which contains a list of available models,
-their versions as well as accepted input and output shapes. Then it generates tensors containing random data with shapes matched
-to the models served by the service. Both the length of the dataset and the workload duration can be specified independently. The
-synthetic data created is then served in a loop iterating over the dataset until the workload length is satisfied. As the main role
-of the client is performance measurement all aspects unrelated to throughput and/or latency are ignored. This means the client does
-not validate the received responses nor does it estimate accuracy as these activities would negatively affect the measured performance
-metrics on the client side.
+The benchmark client introduced in this directory is written in Python 3. Benchmark client uses TFServing API and KServe API to communicate with OpenVINO/TFServing/KServe based services. It is recommended to use the benchmark client as a docker container. Prior to transmission, the client downloads metadata from the server, which contains a list of available models, their versions as well as accepted input and output shapes. Then it generates tensors containing random data with shapes matched to the models served by the service. Both the length of the dataset and the workload duration can be specified independently. The synthetic data created is then served in a loop iterating over the dataset until the workload length is satisfied. As the main role of the client is performance measurement all aspects unrelated to throughput and/or latency are ignored. This means the client does not validate the received responses nor does it estimate accuracy as these activities would negatively affect the measured performance metrics on the client side.
```

Comment:
I would probably change:
`OpenVINO/TFServing/KServe based services` to something like: `inference services` or `model servers`.
I get a feeling that with current statement we are mixing terms with different meaning.